python/dgl/nn/pytorch/conv/relgraphconv.py

```diff
@@ -49,16 +49,17 @@ class RelGraphConv(nn.Module):
     out_feat : int
         Output feature size; i.e., the number of dimensions of :math:`h_i^{(l+1)}`.
     num_rels : int
-        Number of relations. .
+        Number of relations.
     regularizer : str, optional
-        Which weight regularizer to use "basis" or "bdd":
+        Which weight regularizer to use ("basis", "bdd" or ``None``):
 
-         - "basis" is short for basis-decomposition.
-         - "bdd" is short for block-diagonal-decomposition.
-
-        Default applies no regularization.
+         - "basis" is for basis-decomposition.
+         - "bdd" is for block-diagonal-decomposition.
+         - ``None`` applies no regularization.
+    
+        Default: ``None``.
     num_bases : int, optional
-        Number of bases. Needed when ``regularizer`` is specified. Default: ``None``.
+        Number of bases. Needed when a regularization is applied. If ``None``, uses number of relations (``num_rels``). Default: ``None``.
```

Comment:
"Needed when a regularization is applied." -> "It comes into effect when a regularizer is applied." to avoid the ambiguity between "needed to provide by users" and "needed by the regularizer".

Reply:
Thanks. I updated.